app/src/main/java/com/epfl/beatlink/MainActivity.kt

```diff
@@ -102,8 +103,8 @@ fun BeatLinkApp() {
   val mapViewModel: MapViewModel =
       viewModel(factory = MapViewModel.provideFactory(mapLocationRepository))
 
-  NavHost(navController = navController, startDestination = Route.WELCOME) {
-    navigation(startDestination = Screen.WELCOME, route = Route.WELCOME) {
+  NavHost(navController = navController, startDestination = Route.HOME) {
+    navigation(startDestination = Screen.HOME, route = Route.HOME) {
```

Comment:
I think you might have changed this for testing purposes, but the previous version was correct. With the current implementation, we can’t access the welcome, login, and register screens.

```suggestion
  NavHost(navController = navController, startDestination = Route.WELCOME) {
    navigation(startDestination = Screen.WELCOME, route = Route.WELCOME) {
```